_pages/faqs.html

```diff
@@ -105,9 +105,9 @@ <h2 class="text-center">Opportunity Grants</h2>
     <div class="row column faq-list">
       <section class="faq-item">
         <h2 class="text-center">Sponsors</h2>
-        <p><em>More information is on our <a href="{% link _pages/sponsors-information.md %}">Become a Sponsor</a> page. Sponsorship signups for 2022 are now closed.</em></p>
+        <p><em>More information is on our <a href="{% link _pages/sponsors-information.md %}">Become a Sponsor</a> page. Sponsorship signups for 2023 are now closed.</em></p>
```

Comment:
Should we remove the mention that sponsorship signups are closed?

Reply:
Fixed, thanks!